scipy/spatial/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from scipy.spatial.distance import _METRICS_NAMES
+
+
+@pytest.fixture(params=_METRICS_NAMES)
+def metric(request):
```

Comment:
Can you move the test changes to another PR and consider just using `pytest.mark.parametrize` instead of a parametrized fixture?